index.js

```diff
@@ -29,11 +29,81 @@ function enterState(newState) {
 start();
 
 async function start() {
-  const welcomeMessage = `182 Main St.
-You are standing on Main Street between Church and South Winooski.
+  const welcomeMessage = `109 Ontario Street
+You are standing in a haunting wooded area at dusk. You hear crows cawing and fluttering 
+their wings above. To the left there is a lantern. To the right is a bench. Behind you is a
+church.' 
 There is a door here. A keypad sits on the handle.
 On the door is a handwritten sign.`;
   let answer = await ask(welcomeMessage);
-  console.log('Now write your code to make this work!');
-  process.exit();
+  console.log('We Offer Sanctuary to Those in Need');
+  process.exit('Sanctuary');
+  break;
+  default;
+  console.log('Sorry, you can not enter.');
+  break;
 }
+{
+    console.log('You have sanctuary and are in the narthex. Now you must check-in with the residential 
+    ghost priest. 
+    This church has long been abandoned. Behind you, above the door you just entered, is a 
+    stained glass rose window. In front of you is the nave. There is the left and right transept. 
+    All the way to the front is the alter.')
+        {
+            switch (prompt('Please go check in.')){
+                case "behind":
+                console.log('The ghost priest is not there.');
+                case "Left";
+                console.log('The ghost is not there.');
+                case "right";
+                console.log('Nope, not there.'); 
+                case "alter";
+                console.log('Yes, the priest is at the alter.');
+                // Do I have to use " or ' for both areas of text? Does it need to be consistent??
+            }
```

Comment:
A switch statement is used to evaluate a variable, and execute different chunks of code based on the value of that variable. This isn't quite the right usage of them, but it is close. First of all there is no `prompt` function in this code, so you won't be able to use it in this program. The variable we actually want to check is the one which stores user input, in this case it's `answer`. Also you will need colons, not semi-colons after your `case` statements
```js
switch (answer) {
                case "behind":
                    console.log('The ghost priest is not there.');
                case "Left";
                    console.log('The ghost is not there.');
                case "right":
                    console.log('Nope, not there.'); 
                case "alter":
                    console.log('Yes, the priest is at the alter.');
            }
```
The way this reads is: if answer === "behind": console.log('The ghost priest is not there.'); else if answer === "Left": console.log('The ghost is not there.'); etc.  You could also use `if ... else` logic statements to determine your game logic, and while it's a little more verbose it's also a little clearer about whats going on.